remc2/sub_main.cpp

```diff
@@ -117930,47 +117920,12 @@ int sub_84000(int a1)//265000
 	return result;
 }
 
-//----- (00084050) --------------------------------------------------------
-void sub_84050()//265050
+void GetSystemSpeed()
 {
-	//char v0; // sf
-	//int result; // eax
-	//char v2; // [esp+0h] [ebp-8h]
-	//char v3; // [esp+1h] [ebp-7h]
-	//char v4; // [esp+2h] [ebp-6h]
-	//char v5; // [esp+3h] [ebp-5h]
-	//char v6; // [esp+4h] [ebp-4h]
-
-	/*
-	//fix it
-	v0 = 0;
-	//fix it
-
-	sub_840B8(0, 0, v0, 0);
-	v2 = x_BYTE_E29EC;
-	v4 = x_BYTE_E29ED;
-	v5 = x_BYTE_E29EE;
-	v6 = x_BYTE_E29EF;
-	v3 = x_BYTE_E29F0;*/
-
-	//fix computer detect speed
-#ifdef FIX_DETECT_SPEED
-	x_BYTE_E29EC = 5;
-#endif FIX_DETECT_SPEED
-	//fix computer detect speed
-
-	if (x_BYTE_E29EC == 5)
-	{
-		//result = (int)x_D41A0_BYTEARRAY_4;
-		x_D41A0_BYTEARRAY_4_struct.byteindex_34 = 1;
-	}
-	else
-	{
-		//result = (int)x_D41A0_BYTEARRAY_4;
-		x_D41A0_BYTEARRAY_4_struct.byteindex_34 = 0;
-	}
-	//return result;
+	//Just assume High spec for now
+	x_D41A0_BYTEARRAY_4_struct.m_wHighSpeedSystem = 5;
```

Comment:
Makes sense. There is also some code for reading dig.ini and midi.ini as well as dealing with IRQ settings etc. that can just be removed in my opinion.
Moreover, what do you think of removing the 320 graphics mode at all?